platform/linux-generic/Makefile.am

```diff
@@ -195,22 +195,6 @@ noinst_HEADERS = \
 		  ${srcdir}/include/protocols/thash.h \
 		  ${srcdir}/include/protocols/udp.h
 
-if ARCH_IS_ARM
-noinst_HEADERS += ${srcdir}/arch/arm/odp_atomic.h \
-		  ${srcdir}/arch/arm/odp_cpu.h \
-		  ${srcdir}/arch/arm/odp_cpu_idling.h \
-		  ${srcdir}/arch/arm/odp_llsc.h
```

Comment:
These headers should go to some ARCH_IS_ARM-bound variable. Otherwise they will not be included into distribution.